### PR TITLE
Added ability to unflip text

### PR DESCRIPTION
**Pull request by adamrmelnyk**
- Added method for flipping characters
  - does a lookup for a key if there is no value => key
  - Added test
  - bumped version for patch

### CODE REVIEW

File: lib/lita/handlers/flip.rb [hunk 1]
```diff
@@ -75,9 +75,15 @@ def flip_sentence(sentence)
 
       def flip_word(word)
         FLIPPED_WORDS.fetch(word) do |w|
-          w.chars.map { |char| FLIPPED_CHARACTERS[char] }.join.reverse
+          w.chars.map { |char| flip_char(char) }.join.reverse
         end
       end
+
+      def flip_char(char)
+        flipped_char = FLIPPED_CHARACTERS[char] unless flipped_char
```
Comment (zacstewart) [inline lib/lita/handlers/flip.rb]: Would something like this do what you want?

```ruby
FLIPPED_CHARACTERS.fetch(char) { FLIPPED_CHARACTERS.key(char) }
```

Comment (adamrmelnyk) [inline lib/lita/handlers/flip.rb]: Much better looking. I'll make that change.

**Comment by adamrmelnyk** [comment]
I'm not sure about the failed check from semaphore. Never used it and I'm not sure if it's applicable for gems.

**Comment by zacstewart** [comment]
Yeah. Looks like Semaphore screwed up somewhere.

**Pull request merged**